mobile/ios/fastlane/Fastfile

```diff
@@ -32,6 +32,14 @@ platform :ios do
     )
   end
   
+  # Helper method to get version from pubspec.yaml
+  def get_version_from_pubspec
+    pubspec_path = File.join(Dir.pwd, "../..", "pubspec.yaml")
+    pubspec_content = File.read(pubspec_path)
+    version_match = pubspec_content.match(/^version:\s+(.+?)\+/)
+    version_match ? version_match[1] : nil
+  end
```

Comment:
Thoughts on using actual yaml parsing here? Should be part of the ruby stdlib. Something like

```suggestion
def get_version_from_pubspec
  require 'yaml'
  
  pubspec_path = File.join(Dir.pwd, "../..", "pubspec.yaml")
  pubspec = YAML.load_file(pubspec_path)
  
  version_string = pubspec['version']
  version_string ? version_string.split('+').first : nil
end
```